app/modules/backup/continuous-replication/view.jade

```diff
@@ -70,6 +70,7 @@ form#ciform(ng-submit = 'ctrl.save(ctrl.formData.scheduleId, ctrl.formData.selec
           label.control-label.col-md-2(for = 'enabled')
             input#enabled(form = 'ciform', ng-model = 'ctrl.formData.enabled', type = 'checkbox')
           .help-block.col-md-8 Enable immediately after creation
+        include ../includes/reportWhen.jade
```

Comment:
We have not used includes for now, let's not start now, especially for a so small change :)